code/game/gamemodes/miniantags/bot_swarm/swarmer.dm

```diff
@@ -306,6 +306,11 @@
 /obj/spacepod/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
 	to_chat(S, "<span class='warning'>Destroying this vehicle would destroy us. Aborting.</span>")
 
+/obj/machinery/clonepod/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
+	if(attempting)
+		to_chat(S, "<span class='warning'>Destroying this machine while an it is occupied would result in biological and sentient resources to be harmed. Aborting.</span>")
```

Comment:
Remove the `an` from `while an it is occupied `

Reply:
errr,dumb me

Reply:
Grammer addressed